File: src/com/l2kt/gameserver/model/actor/instance/RiftInvader.java
package com.l2kt.gameserver.model.actor.instance;

import com.l2kt.gameserver.model.actor.template.NpcTemplate;

public class RiftInvader extends Monster
{
	// Not longer needed since rift monster targeting control now is handled by the room zones for any mob
	public RiftInvader(int objectId, NpcTemplate template)
	{
		super(objectId, template);
	}
}

File: src/com/l2kt/gameserver/model/actor/instance/RiftInvader.kt
package com.l2kt.gameserver.model.actor.instance

import com.l2kt.gameserver.model.actor.template.NpcTemplate

// Not longer needed since rift monster targeting control now is handled by the room zones for any mob
class RiftInvader(objectId: Int, template: NpcTemplate) : Monster(objectId, template)
